website/markdown/docs/commands/signing.mdx

```diff
@@ -24,18 +24,18 @@ Once you download them, make sure to rename the files, so they abide by the foll
 Where `Target` should be a name of the target, `Configuration` a name of the configuration and leave the default extension
 that you downloaded the file with (eg. `p12` for certificate, etc.).
 Now you can put all those files in `Tuist/Signing` directory.
-To make it all work, create a secure password and place its contents into `Tuist/master.key` that will be used
+To make it all work, create a secure password by running [tuist secret](/docs/commands/secret/) and place its contents into `Tuist/master.key` that will be used
 for encrypting and decrypting your files.
 
 After that, feel free to run `tuist generate` and everything else should be done for you -
 that means, yes, also build settings!
 
 ## Additional commands
 
-`tuist generate` automatically encrypts all signing artifacts.
+`tuist signing generate` automatically encrypts all signing artifacts.
```

Comment:
I believe the `tuist generate` was correct, actually, there is no `tuist signing generate` 🙂

Reply:
Of course, it was 🤦‍♂️